src/test/java/com/microsoft/sqlserver/jdbc/resiliency/ResultSetsWithResiliencyTest.java

```diff
@@ -92,6 +92,7 @@ public void testFullBufferingWithPartiallyParsedResultSet() throws SQLException
                 fail();
             }
         } catch (SQLServerException e) {
+            e.printStackTrace();

```

Comment:
can we not have these in tests? As you "expect" these exceptions, they unnecessarily will show up in pipelines.

Reply:
I think this was added for debugging before. yes, please remove